Gruntfile.js

```diff
@@ -135,11 +135,10 @@ grunt.initConfig({
 		}
 	},
 
-	// Remove the requireSpacesInsideParentheses override once everything is fixed
 	jscs: {
 		all: {
 			options: {
-				requireCapitalizedComments: null
+				config: ".jscsrc"
```

Comment:
I assume you meant to comment on the max errors line above but yeah its not a bug its just the default is 50.